src/containers/Projects/CreateProjectPopup/index.js

```diff
@@ -241,6 +242,7 @@ let CreateProjectPopup = (props) => {
             name="description"
             component={TextareaField}
             validate={[required]}
+            autoComplete="new-password"
```

Comment:
Same here

Reply:
this is also being used as auto-complete=off  now as auto-complete=off is not working for chrome any more